CONTRIBUTING.md

```diff
@@ -1,66 +1,43 @@
 # Contributing to Kangal
 
-:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
-
-The following is a set of guidelines for contributing to Kangal and its packages, 
-which are hosted in the [HelloFresh Organization](https://github.com/hellofresh) on GitHub.
-These are just guidelines, not rules. Use your best judgment, and feel free to propose changes 
-to this document in a pull request.
+:+1::tada: First off all, thanks for taking the time to contribute! :tada::+1:
+
+The project is currently in the maintenance mode, which means we don't actively implement new features.
+We accept the external contribution after the discussions in [GitHub issues](https://github.com/hellofresh/kangal/issues).
+
+Kangal is currently maintained by [HelloFresh](https://github.com/hellofresh).
+You can also reach to us by email [engineering-experience@hellofresh.com](mailto:engineering-experience@hellofresh.com)
```

Comment:
I would remove the email from here and other places. Better to centralize communication through github issues in the project.

Reply:
removed!